docs/sources/administration/provisioning/index.md

```diff
@@ -615,13 +615,18 @@ The following sections detail the supported settings and secure settings for eac
 
 #### Alert notification `webhook`
 
+<!-- vale Grafana.Spelling = NO -->
+
 | Name        | Secure setting |
 | ----------- | -------------- |
 | url         |                |
 | http_method |                |
 | username    |                |
 | password    | yes            |
 | tls_config  |                |
+| hmac_config |                |
+
+<!-- vale Grafana.Spelling = YES -->
```

Comment:
Configuration settings should be in `code`: https://grafana.com/docs/writers-toolkit/write/style-guide/style-conventions/#code


```suggestion
| Name          | Secure setting |
| ------------- | -------------- |
| `url`         |                |
| `http_method` |                |
| `username`    |                |
| `password`    | yes            |
| `tls_config`  |                |
| `hmac_config` |                |
```